DSharpPlus.Commands/Processors/SlashCommands/SlashCommandProcessor.cs

```diff
@@ -36,20 +36,31 @@ public sealed partial class SlashCommandProcessor : BaseCommandProcessor<Interac
     public IReadOnlyDictionary<ulong, Command> ApplicationCommandMapping => applicationCommandMapping;
 
     /// <inheritdoc/>
-    public override IReadOnlyList<Command> Commands => this.ApplicationCommandMapping.Values.ToList(); //TODO: alloc free?
+    public override IReadOnlyList<Command> Commands => [.. this.ApplicationCommandMapping.Values];
+
+    /// <summary>
+    /// The configuration values being used for this processor.
+    /// </summary>
+    public SlashCommandConfiguration Configuration { get; init; }
 
     private static readonly List<DiscordApplicationCommand> applicationCommands = [];
     private static FrozenDictionary<ulong, Command> applicationCommandMapping = FrozenDictionary<ulong, Command>.Empty;
 
     [GeneratedRegex(@"^[-_\p{L}\p{N}\p{IsDevanagari}\p{IsThai}]{1,32}$")]
-    private partial Regex NameLocalizationRegex();
+    private static partial Regex NameLocalizationRegex();
 
     [GeneratedRegex("^.{1,100}$")]
-    private partial Regex DescrtiptionLocalizationRegex();
+    private static partial Regex DescrtiptionLocalizationRegex();

```

Comment:
Is there any particular benefit to making these static?

Reply:
marginally, but it doesn't hurt either